cookiecutter/exceptions.py

```diff
@@ -113,3 +113,26 @@ class RepositoryNotFound(CookiecutterException):
     """
     Raised when the specified cookiecutter repository doesn't exist.
     """
+
+
+class RepositoryCloneFailed(CookiecutterException):
+    """Raised when a cookiecutter template can't be cloned."""
+
+    BRANCH_ERRORS = [
+        'error: pathspec',
+        'unknown revision',
+    ]
+
+    def __init__(self, exc, repo_url, checkout):
```

Comment:
``exc`` is somewhat cryptic. How do you feel about changing this to ``exception`` or ``error``?